cmd/soroban-cli/src/config/alias.rs

```diff
@@ -1,8 +1,49 @@
-use std::collections::HashMap;
+use std::{collections::HashMap, convert::Infallible, str::FromStr};
 
 use serde::{Deserialize, Serialize};
 
+use super::locator;
+
 #[derive(Serialize, Deserialize, Default)]
 pub struct Data {
     pub ids: HashMap<String, String>,
 }
+
+/// Address can be either a contract address, C.. or eventually an alias of a contract address.
+#[derive(Clone, Debug)]
+pub enum ContractAddress {
+    ContractId(stellar_strkey::Contract),
+    Alias(String),
+}
+
+impl Default for ContractAddress {
```

Comment:
Does ContractAddress need to have a default? 🤔

Reply:
Good question I copy/pasted from address::Address. I'll look into removing it.

Reply:
Checked and some of the commands that use it require it.